CONTRIBUTING.md

```diff
@@ -910,6 +910,23 @@ or
 QUICK_BUILD=true CONTAINER_IMAGE_REPO=<your name>/vector-test make test-e2e-kubernetes
 ```
 
+#### Kubernetes Architecture
```

Comment:
Does this need to be added to the index at the top of the file?

Reply:
There's a MarkdownTOC annotation. Do you happen to know how do I trigger the regeneration? I used a VS Code plugin for things like these before, but it's broken atm.

Reply:
It's a sublime text plugin :(. I would just manually add it for now.